.tool/lint

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+for d in $(find . -type d -not -iwholename '*.git*' -a -not -iname '.tool' -a -not -iwholename '*vendor*'); do
```

Comment:
Theirs also need to be updated when we include new non-go directories, right?

The only golang package I could image we'll add is `test` directory. WDYT?